examples/compose.js

```diff
@@ -1,23 +1,21 @@
-import merge from 'lodash/object/merge';
-import map from 'lodash/collection/map';
-import isUndefined from 'lodash/lang/isUndefined';
-import isFunction from 'lodash/lang/isFunction';
+import merge from 'lodash/merge';
+const isFunction = (obj) => typeof obj === 'function';
+const assign = Object.assign;
 
 const getDescriptorProps = (descriptorName, composables) => {
-  return map(composables, composable => {
+  return !composables ? undefined : composables.map(composable => {
```

Comment:
`null` is fine too. I just was trying to mimic the `lodash.map` behavior.
Change it to `null`?